src/GitLink/Providers/VisualStudioTeamServicesProvider.cs

```diff
@@ -44,6 +44,8 @@ public override bool Initialize(string url)
                 ProjectName = match.Groups["repo"].Value;
             }
 
+            RepositoryName = match.Groups["repo"].Value;
```

Comment:
So actually, the reason I added the RepositoryName property was because in VSTS the concept of a repository is separate from a project. You can have multiple repositories within a project. The RepositoryName property is only effectively used in Linker.cs.